app/views/patients/monitoree_unavailable.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-24 d-flex flex-column justify-content-center">
+      <%= render partial: 'errors/logo' %>
+      <div class="col-md-15 col-lg-12 col-xl-10 mx-auto">
+        <div class="card">
+          <div class="card-header">
+            <h4 class="text-center m-0">Monitoree Unavailable</h4>
```

Comment:
Small tweak to make it look more like the mockup
```suggestion
          <div class="card-header bg-primary">
            <h4 class="text-center m-0 text-white">Monitoree Unavailable</h4>/h4>
```

Reply:
so I don't love this as much because it takes away from the button, but will run both by the PO team to see which they prefer
<img width="749" alt="Screen Shot 2021-04-21 at 5 24 17 PM" src="https://user-images.githubusercontent.com/35042815/115623176-d5418f80-a2c6-11eb-946f-a528f5f05d54.png">

Reply:
Fair point. I don't have a strong preference personally, was just going off the mockup. It also has alignment left as opposed to center, but I definitely like center more.

Reply:
I definitely agree with you on that.  Let's see what the PO team comes back with for the card header

Reply:
swapped to blue: https://github.com/SaraAlert/SaraAlert/pull/736/commits/24eb9e95f3781d1067cbb86d3926b45d6e3fc1c0